uiowa/uiowa
# Update UIDS and implement component changes to fix card/banner contextual link issue.

Resolves #2277

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
```
Test on 

- https://sandbox.prod.drupal.uiowa.edu/cards
- https://sandbox.prod.drupal.uiowa.edu/banners-and-buttons
- Verify that `click-container` appears on wrapping div of `card` and `banners`.
- If `card` and `banners` have one link, verify that link class of `click-target` appears on heading `<a>`.
- Verify that you can edit card media when logged in and on card page. 
- Verify that pause and link work on https://sandbox.local.drupal.uiowa.edu/ banner video loop.
